### DISCUSSION TITLE
Recently played should collapse albums if the album was selected to be played

### DISCUSSION THREAD

**Discussion by JeffersonBledsoe** [music-assistant/support#1945] [Feature requests and ideas]
The "Recently played" seems to display every track within an album, even if I pressed "Play album". It would be nice if we could more accurately reflect how somebody chose to play their music in the recently played (and reduce noise whilst we're at it).

Tested with Spotify Music Provider and HA Player on MA b103

**Comment by OzGav** [comment]
This has been implemented
